agent-chat/public/app.css

```
@@ -88,13 +125,27 @@ body {
 .provider-icon-img {
   display: inline-block; background: center / contain no-repeat;
 }
+.provider-running .provider-icon,
+.provider-running .provider-icon-img { animation: harnessPulse 2s ease-in-out infinite; }
+@keyframes harnessPulse {
```

Stylelint (17.14.0)

[error] Expected keyframe name "harnessPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)